src/content/legal/vilkar.md

```diff
@@ -109,9 +109,15 @@ Tjenesten krever:
 
 ## 8. Oppsigelse og avslutning
 
-### 8.1 Kundens oppsigelse
+### 8.1 Kundens oppsigelse og oppsigelsesvilkår
 Du kan avslutte bruken av tjenesten når som helst ved skriftlig varsel via vårt [kontaktskjema](/kontakt).
 
+Ved oppsigelse gjelder følgende:
+- Du kan si opp abonnementet når som helst uten bindingstid
+- Oppsigelsen trer i kraft umiddelbart og tilgangen til tjenesten avsluttes
+- Ingen refusjon gis for den gjenværende delen av den betalte perioden
+- Eksempel: Hvis du sier opp 15. mars og har betalt for hele mars måned, mister du umiddelbart tilgang til tjenesten uten refusjon for de siste 16 dagene
```

Comment:
Ettersom vi ikke har måned for måned men år for år fakturering, så må eksempelet gjenspeile dette